atomic_reactor/util.py

```diff
@@ -1454,30 +1454,21 @@ def get_log_files(self, osbs, build_id):
 
         # Collect logs from server
         try:
-            logs = osbs.get_build_logs(build_id)
+            logs = osbs.get_build_logs(pipeline_run_name)
         except OsbsException as ex:
             self.log.error("unable to get build logs: %s", ex)
             return output
-        except TypeError:
-            # Older osbs-client has no get_orchestrator_build_logs
-            self.log.error("OSBS client does not support get_orchestrator_build_logs")
-            return output
 
-        platform_logs = {}
-        for entry in logs:
-            platform = entry.platform
-            if platform not in platform_logs:
-                filename = 'orchestrator' if platform is None else platform
-                platform_logs[platform] = NamedTemporaryFile(prefix="%s-%s" %
-                                                             (build_id, filename),
-                                                             suffix=".log", mode='r+b')
-            platform_logs[platform].write((entry.line + '\n').encode('utf-8'))
-
-        for platform, logfile in platform_logs.items():
-            logfile.flush()
-            filename = 'orchestrator' if platform is None else platform
-            metadata = self.get_log_metadata(logfile.name, "%s.log" % filename)
-            output.append(Output(file=logfile, metadata=metadata))
+        # OSBS2 TBD logs for specific arches for binary builds
+        filename = 'orchestrator'
```

Comment:
rename this log to 'osbs_build.log' like in https://github.com/containerbuildsystem/koji-containerbuild/blob/master/koji_containerbuild/plugins/builder_containerbuild.py#L309

Reply:
done